### ISSUE TITLE
Rename counters to indicate position counted from

### ISSUE THREAD

**Issue by robgjansen** [privcount/privcount#228]
Would it be worth it to rename all of the counters so that they indicate the position the counter is counted from? Is this as easy as a simple find/replace across the project, or is it more complicated than that?

----

These are the entry counters (`cat privcount/data_collector.py  | grep -e "Entry" -e "CircuitCells" -e "ClientIP" | cut -d'"' -f2 | sort | grep -v "#"`):

```
CircuitCellsIn
CircuitCellsOut
CircuitCellsRatio
CircuitsActiveEntry
CircuitsAllEntry
CircuitsInactiveEntry
ClientIPCircuitsActive
ClientIPCircuitsInactive
ClientIPsActive
ClientIPsInactive
ClientIPsUnique
```

I propose a rename to:

```
EntryCircuitCellsIn
EntryCircuitCellsOut
EntryCircuitCellsRatio
EntryCircuitsActive
EntryCircuitsAll
EntryCircuitsInactive
EntryClientIPCircuitsActive
EntryClientIPCircuitsInactive
EntryClientIPsActive
EntryClientIPsInactive
EntryClientIPsUnique
```

----

These are the exit counters (`cat privcount/data_collector.py  | grep -e "Stream" -e "Circuit" -e "Sanity" -e "ClientIP" | cut -d'"' -f2 | sort | grep -v "#" | grep -v "Entry" | grep -v "CircuitCells" | grep -v "ClientIP"`):

```
CircuitInterStreamCreationTime
CircuitInterStreamCreationTimeInteractive
CircuitInterStreamCreationTimeOther
CircuitInterStreamCreationTimeP2P
CircuitInterStreamCreationTimeWeb
CircuitLifeTimeActive
CircuitLifeTimeAll
CircuitLifeTimeInactive
CircuitsActive
CircuitsAll
CircuitsInactive
CircuitsInteractive
CircuitsOther
CircuitsP2P
CircuitStreamsAll
CircuitStreamsInteractive
CircuitStreamsOther
CircuitStreamsP2P
CircuitStreamsWeb
CircuitsWeb
StreamBytesAll
StreamBytesInAll
StreamBytesInInteractive
StreamBytesInOther
StreamBytesInP2P
StreamBytesInteractive
StreamBytesInWeb
StreamBytesOther
StreamBytesOutAll
StreamBytesOutInteractive
StreamBytesOutOther
StreamBytesOutP2P
StreamBytesOutWeb
StreamBytesP2P
StreamBytesRatioAll
StreamBytesRatioInteractive
StreamBytesRatioOther
StreamBytesRatioP2P
StreamBytesRatioWeb
StreamBytesWeb
StreamLifeTimeInteractive
StreamLifeTimeOther
StreamLifeTimeP2P
StreamLifeTimeWeb
StreamsAll
StreamsInteractive
StreamsOther
StreamsP2P
StreamsWeb
```

I propose a rename to:

```
ExitCircuitInterStreamCreationTime
ExitCircuitInterStreamCreationTimeInteractive
ExitCircuitInterStreamCreationTimeOther
ExitCircuitInterStreamCreationTimeP2P
ExitCircuitInterStreamCreationTimeWeb
ExitCircuitLifeTimeActive
ExitCircuitLifeTimeAll
ExitCircuitLifeTimeInactive
ExitCircuitsActive
ExitCircuitsAll
ExitCircuitsInactive
ExitCircuitsInteractive
ExitCircuitsOther
ExitCircuitsP2P
ExitCircuitStreamsAll
ExitCircuitStreamsInteractive
ExitCircuitStreamsOther
ExitCircuitStreamsP2P
ExitCircuitStreamsWeb
ExitCircuitsWeb
ExitStreamBytesAll
ExitStreamBytesInAll
ExitStreamBytesInInteractive
ExitStreamBytesInOther
ExitStreamBytesInP2P
ExitStreamBytesInteractive
ExitStreamBytesInWeb
ExitStreamBytesOther
ExitStreamBytesOutAll
ExitStreamBytesOutInteractive
ExitStreamBytesOutOther
ExitStreamBytesOutP2P
ExitStreamBytesOutWeb
ExitStreamBytesP2P
ExitStreamBytesRatioAll
ExitStreamBytesRatioInteractive
ExitStreamBytesRatioOther
ExitStreamBytesRatioP2P
ExitStreamBytesRatioWeb
ExitStreamBytesWeb
ExitStreamLifeTimeInteractive
ExitStreamLifeTimeOther
ExitStreamLifeTimeP2P
ExitStreamLifeTimeWeb
ExitStreamsAll
ExitStreamsInteractive
ExitStreamsOther
ExitStreamsP2P
ExitStreamsWeb
```

**Comment by teor2345** [comment]
You're missing some counters:
* Connection*
  * we collect these everywhere (no prefix?)
  * see #110 for corrected greps
* SanityCheck
  * we collect this everywhere (no prefix?)
* traffic model
  * this only works on Exits, because it needs streams
  * should they have an Exit prefix, 

In principle, I'm ok with this.

And it should be a simple search and replace, except for prefix matches:
```
CircuitInterStreamCreationTime
CircuitInterStreamCreationTimeInteractive
CircuitInterStreamCreationTimeOther
CircuitInterStreamCreationTimeP2P
CircuitInterStreamCreationTimeWeb
```
I'd suggest you change CircuitInterStreamCreationTime to CircuitInterStreamCreationTimeAll, then do the global replacement.

**Comment by teor2345** [comment]
ConnectionsAll
    ConnectionLifeTime
to
    EntryConnectionsAll
    EntryConnectionLifeTime

And I'm going to rename using a consistent format:
<Entry/Exit/><Bytes/Stream/Circuit/CircuitStream/Connection><Object/Qualifier>*<Count/Ratio/Time>

**Comment by teor2345** [comment]
Fixed by pull #265.
